packages/kms-musap-rn/README.md

```diff
@@ -0,0 +1,16 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>Sphereon's Musap Key Management System for react-native
+  <br>
+</h1>
+
+This Sphereon KMS wrapper module enables the use of MUSAP KMS functionalities within React Native projects. It extends the capabilities of the [`AbstractKeyManagementSystem`](./node_modules/@veramo/key-manager/src/abstract-key-management-system.ts) by integrating the MUSAP library to align with Veramo's key management functions.
+A Sphereon KMS wrapper implementation that provides functionalities of musap kms for react-native projects.
```

Comment:
Please make this a proper readme. There will be a jury with technical people looking at these deliverables

Reply:
changed the readme to have necessary guides for all the functions